remo/events/views.py

```diff
@@ -37,10 +38,11 @@ def redirect_list_events(request):
 @cache_control(private=True)
 def list_events(request):
     """List events view."""
-    events = Event.objects.all()
-    categories = FunctionalArea.objects.all()
-    return render(request, 'list_events.html',
-                  {'events': events, 'categories': categories})
+    data = {'events': Event.objects.all(),
+            'categories': FunctionalArea.objects.all(),
+            'initiatives': Campaign.objects.all()}
```

Comment:
API returns only active ``campaigns``. We might need to consider this as well here.